webapp/channels/src/components/textbox/textbox_links.tsx

```diff
@@ -4,15 +4,15 @@
 import React from 'react';
 import type {MouseEvent} from 'react';
 import {FormattedMessage} from 'react-intl';
-import {Link} from 'react-router-dom';
+
+import ExternalLink from '../external_link';
```

Comment:
Mind changing that to an absolute import? We usually avoid relative ones which require a `..`

Reply:
Fixed!